From 10aa67a0cab480c9df2a050e0d102aca0cf18a02 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Fri, 27 Nov 2020 15:16:57 +0000
Subject: [PATCH] Revert "set CARGO_HOME to $SRCDEST/cargo"

the SRCDEST may be shared by multiple builders and apparently cargo does
not seem to handle that well. There are also no good way to do clean-up
of the cache for purging things that is no longer in use.

This reverts commit af0c88e6abbb1e49224759f5c51b3068e6eab28b.
---
 abuild.in | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/abuild.in b/abuild.in
index d600e92..eeb5f62 100644
--- a/abuild.in
+++ b/abuild.in
@@ -2685,7 +2685,6 @@ aportsgit=${APORTSDIR:-${startdir}}
 repo_template=$aportsgit/$repo/.rootbld-repositories
 
 SRCDEST=${SRCDEST:-$startdir}
-CARGO_HOME=$SRCDEST/cargo
 
 BUILD_ROOT=
 
@@ -2704,7 +2703,7 @@ if [ -z "$REPODEST" ]; then
 fi
 
 # for recursive action
-export REPODEST SRCDEST CARGO_HOME
+export REPODEST SRCDEST
 
 # add dbg subpackage if its enabled globally
 if [ -n "$DEFAULT_DBG" ] && ! subpackage_types_has "dbg" && ! options_has "!dbg" && [ "$arch" != "noarch" ]; then
-- 
2.29.2

